pl.suve.colorful.yaml

```diff
@@ -0,0 +1,75 @@
+app-id: pl.suve.colorful
+
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.freepascal
+
+build-options:
+  append-path: /usr/lib/sdk/freepascal/bin
+  env:
+    PPC_CONFIG_PATH: /usr/lib/sdk/freepascal/etc
+    FPCDIR: /usr/lib/sdk/freepascal/fpcsrc
+
+command: colorful
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --device=dri
+
+modules:
+  - name: optipng
```

Comment:
of this is build time only, a cleanup rule of `*` is missing.

Reply:
Thanks, fixed in 5659b31a8bc1afbd02b30ac6a324af6067f28c77.